docs/superpowers/plans/2026-09-05-p10-declared-pre-press-clean.md

```
@@ -0,0 +1,1082 @@
+    result = execute_clean(command, target)
+    sys.stdout.write(result.stdout.decode("utf-8", "replace"))
+    if result.returncode != 0:
+        sys.stderr.write(result.stderr.decode("utf-8", "replace"))
+        print(f"error: git clean exited {result.returncode}", file=sys.stderr)
+        return 1
+    return 0
+```
+
+`press_cli.py` — import `clean_cli` alongside `check_tools`, add the usage line between `verify` and `check-tools`:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)